pkg/virtctl/vm/evacuate_cancel.go

```diff
@@ -0,0 +1,144 @@
+package vm
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	virtv1 "kubevirt.io/api/core/v1"
+	"kubevirt.io/client-go/kubecli"
+
+	"kubevirt.io/kubevirt/pkg/virtctl/clientconfig"
+	"kubevirt.io/kubevirt/pkg/virtctl/templates"
+)
+
+func NewEvacuateCancelCommand() *cobra.Command {
+	c := &evacuateCancelCommand{}
+
+	cmd := &cobra.Command{
+		Use:     "evacuate-cancel (vm <vm-name> | vmi <vmi-name> | node <node-name>)",
+		Short:   "Cancel evacuation for a VM, VMI, or all VMIs on a node",
+		Example: usageEvacuateCancel(),
+		Args:    cobra.ExactArgs(2),
+		RunE:    c.Run,
+	}
+
+	c.AddFlags(cmd.Flags())
+	cmd.Flags().BoolVar(&dryRun, dryRunArg, false, dryRunCommandUsage)
+	cmd.SetUsageTemplate(templates.UsageTemplate())
+
+	return cmd
+}
+
+func usageEvacuateCancel() string {
+	return `  # Cancel evacuation for a virtual machine
+  {{ProgramName}} evacuate-cancel vm my-vm
+
+  # Cancel evacuation for a virtual machine instance
+  {{ProgramName}} evacuate-cancel vmi my-vmi
+
+  # Cancel evacuation and also cancel an active migration for the VMI
+  {{ProgramName}} evacuate-cancel vmi my-vmi --migrate-cancel
+
+  # Cancel evacuation for all VMIs on a specific node
+  {{ProgramName}} evacuate-cancel node node01
+  `
+}
+
+type evacuateCancelCommand struct {
+	MigrateCancel bool
+	cmd           *cobra.Command
+	virtClient    kubecli.KubevirtClient
+}
+
+func (c *evacuateCancelCommand) AddFlags(fs *pflag.FlagSet) {
+	fs.BoolVar(&c.MigrateCancel, "migrate-cancel", false, "Also cancel an active migration for the specified VMI")
+}
```

Comment:
This method could be inlined into `NewEvacuateCancelCommand`, it does only a single thing.